specification/communication/data-plane/Sms/preview/2024-12-10-preview/communicationservicessms.json

```diff
@@ -197,6 +329,84 @@
           "type": "string"
         }
       }
+    },
+    "OptOutSendRecipient": {
+      "description": "Represents properties for a single recipient.",
+      "required": [
+        "to"
+      ],
+      "type": "object",
+      "properties": {
+        "to": {
+          "description": "The recipient's phone number (in E.164 format).",
+          "minLength": 1,
+          "type": "string"
+        }
+      }
+    },
+    "OptOutSendRequest": {
+      "description": "Represents properties for a Opt request.",
+      "required": [
+        "from",
+        "recipients"
+      ],
+      "type": "object",
+      "properties": {
+        "from": {
+          "description": "The sender's identifier (typically phone number in E.164 format) that is owned by the authenticated account.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "recipients": {
+          "maxItems": 100,
+          "minItems": 1,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OptOutSendRecipient"
+          }
+        }
+      }
+    },
+    "OptOutSendResponse": {
```

Comment:
Consider dropping the Send in most of the names if you feel like `OptOutResponse`, `OptOutRecipient` etc. is descriptive enough to be understood. I think you mean that ((OptOutSend)Response) means opting out of sending, but I personally trip up when I read SendResponse and SendRecipient next to each other.

Reply:
Renamed the difinitions